rep-2002.rst

```diff
@@ -0,0 +1,136 @@
+REP: 2002
+Title: ROS 2 Rolling Release
+Author: Steven! Ragnarok <steven@openrobotics.org>
+Status: Draft
+Type: Informational
+Content-Type: text/x-rst
+Created: 05-Sep-2019
+Post-History: 05-Sep-2019, 03-Oct-2019, 18-Oct-2019
+
+
+Abstract
+========
+
+This REP describes a rolling ROS 2 distribution which will serve as the target for ongoing development of ROS 2.
+It also describes how future releases of ROS 2 may be produced by taking a snapshot of the rolling distribution to produce the next fixed release of ROS 2. 
+
+
+Motivation
+==========
+
+In order to reduce the workload of Open Robotics staff and individual package maintainers when bootstrapping new ROS releases a rolling ROS distribution will be created to serve as the foundation for subsequent stable ROS distributions.
+
+
+Support status
+==============
+
+The rolling release itself will not be considered a supported platform.
+Distribution syncs will be performed on a regular cadence just like stable distributions but regressions may not block a rolling release sync.
+During the pre-release phase for stable ROS distributions releases in the rolling distribution may be blocked or held back in order to conform to the support requirements of the upcoming stable release.
+
+
+Release process
+===============
+
+Releasing stable ROS distributions from a rolling release
+---------------------------------------------------------
+
+When a new stable release is being prepared, the rolling release will be used as a staging ground to prepare that release.
+Updates and new package releases made to the rolling release will be propagated automatically to the stable release.
+At an announced time prior to the official stable release, the stable distribution will be created and packages with successful builds on the rolling release will be automatically released via bloom into the stable release.
+Once the stable distribution has been created and the automatic bloom process has been run, changes to the rolling release may not be automatically propagated to the stable release although 
```

Comment:
You can drop the `although` grammatically & needs a `.`

Reply:
Thanks for the feedback. That although was leading to a further thought that didn't get transcribed. I've added it now.